backend/routing/users.js

```diff
@@ -30,4 +32,60 @@ const usersController = require('../controllers/users.js')(DAL);
       }
     }
   });
+
+  server.route({
+    method: 'POST',
+    path: '/reset-password',
+    config: {
+      handler: function (request, reply) {
+        let resetToken = utils.newToken();
+        DAL.users.addResetToken(resetToken, request.payload.email).then((response) => {
+          const message = [
+            'Link for reset password: ' + 'http://localhost:4200/new-password/' + resetToken,
+          ].join('\n');
+
+          const mail = {
+            from: '<bizkonect.project@gmail.com>', // sender address
+            to: '<bizkonect.project@gmail.com>', //for test
```

Comment:
it's for testing becouse we have only user with user@user.com password and it isn't valid